index.html

```diff
@@ -54,5 +54,195 @@ <h6>LET'S CONNECT</h6>
+
+          <li class="tags">
+            <a href="" class="tag1"><h6>html</h6> </a
+            ><a href="" class="tag2"><h6>css</h6> </a
+            ><a href="" class="tag3"><h6>javascript</h6> </a>
+          </li>
+          <li class="btn">
+            <a><h6>See Project</h6></a>
+          </li>
+        </ul>
+      </article>
+      <article class="card">
+        <a href="" class="snapshot">
+          <img src="/Worksection Pics/snap2.png" alt="" />
+        </a>
+        <ul class="leftblock2">
+          <li .primarytext>
+            <h2>Multi-Post Stories</h2>
+            <div class="frame">
+              <a href="" class="client"><h6 class="clienttext">Canopy</h6></a>
+              <a href=""><img src="/Worksection Pics/Counter.png" alt="" /></a>
+              <a href="" class="role"><h6 class="roletext">Back End Dev</h6></a>
+              <a href=""><img src="/Worksection Pics/Counter.png" alt="" /></a>
+              <a href="" class="year"><h6 class="yeartext">2015</h6></a>
+            </div>
+          </li>
+
+          <p>
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+          </p>
+
+          <li class="tags">
+            <a href="" class="tag1"><h6>html</h6> </a
+            ><a href="" class="tag2"><h6>css</h6> </a
+            ><a href="" class="tag3"><h6>javascript</h6> </a>
+          </li>
+          <li class="btn">
+            <a><h6>See Project</h6></a>
+          </li>
+        </ul>
+      </article>
+      <article class="card">
+        <a href="" class="snapshot">
+          <img src="/Worksection Pics/snap3.png" alt="" />
+        </a>
+        <ul class="leftblock">
+          <li .primarytext>
+            <h2>Tonic</h2>
+            <div class="frame">
+              <a href="" class="client"><h6 class="clienttext">Canopy</h6></a>
+              <a href=""><img src="/Worksection Pics/Counter.png" alt="" /></a>
+              <a href="" class="role"><h6 class="roletext">Back End Dev</h6></a>
+              <a href=""><img src="/Worksection Pics/Counter.png" alt="" /></a>
+              <a href="" class="year"><h6 class="yeartext">2015</h6></a>
+            </div>
+          </li>
+
+          <p>
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+          </p>
+
+          <li class="tags">
+            <a href="" class="tag1"><h6>html</h6> </a
+            ><a href="" class="tag2"><h6>css</h6> </a
+            ><a href="" class="tag3"><h6>javascript</h6> </a>
+          </li>
+          <li class="btn">
+            <a><h6>See Project</h6></a>
+          </li>
+        </ul>
+      </article>
+      <article class="card">
+        <a href="" class="snapshot">
+          <img src="/Worksection Pics/snap4.png" alt="" />
+        </a>
+        <ul class="leftblock2">
+          <li .primarytext>
+            <h2>Multi-Post Stories</h2>
+            <div class="frame">
```

Comment:
Kindly make sure your about section looks like the Figma design as it was required by the previous reviewer [here](https://github.com/studiosnabeel/Portfolio/pull/3#pullrequestreview-1119292015)

to do that I can suggest you to change  the lines I list below in the style of your classes :grinning: :grinning: :grinning: :grinning: :grinning: 
### Remove from your frame classes 
```
 padding: 30px 0; 

  width: 236px; 
   height: 32px;
```
### change the left of all  arrow classes to the correct place 

` left: px;`

Reply:
Fixed.